docs/rfcs/001-recent-commits.md

```diff
@@ -42,37 +42,16 @@ Each **recent commit** within the recent commits section summarizes that commit'
 * GitHub avatar for both the committer and (if applicable) author. If either do not exist, show a placeholder.
 * The commit message (first line of the commit body) elided if it would be too wide.
 * A relative timestamp indicating how long ago the commit was created.
-* A greyed-out state if the commit is reachable from the remote tracking branch but _not_ from HEAD (meaning, if it has been fetched but not pulled).
```

Comment:
I'm a little conflicted about this one. Do we still want to do this? I feel like the fetched commits would almost always overwhelm the panel. Maybe we could have a `3 fetched` line instead... ?

Reply:
I think it's handy to get a hint what the commits are about before you pull the changes. Like when you still have to finish something and can't pull right now, you already know what will come. Since these are on the remote, they could even be click-able and open a browser window with a diff.

That said, I agree that if you're more than 2-3 commits behind, it buries your local changes too much. So I'm ok to punt on it and leave it for a possible log/history.